src/routes/embedded/auth/auth-email-signup/auth-email-signin.view.tsx

```diff
@@ -29,77 +29,38 @@ export function AuthEmailSigninEmbeddedView() {
 
   // Handlers:
 
-  /*
-  const areButtonsDisabled =
-    authStatus === "unknown" ||
-    authStatus === "loading" ||
-    authStatus === "authLoading" ||
-    !authEmail ||
-    !passwordInput.state;
-  */
-
-  const authStatusRef = useRef(authStatus);
-
-  const handleEmailSignin = useCallback(
+  const handleEmailSignIn = useCallback(
     async (e: React.FormEvent) => {
       e.preventDefault();
 
-      const password = passwordInputRef.current?.value || "";
-
       try {
+        const password = passwordInputRef.current?.value || "";
+
         if (!password) {
           toast.error("Please enter an email and password");
           return;
         }
 
         setIsAuthenticating(true);
 
-        // TODO: Call authenticate instead.
-
-        const supabase = await getSupabaseClient();
-
-        const { error, data } = await supabase.auth.signInWithPassword({
+        await authenticate({
+          method: "signInWithPassword",
           email,
           password,
         });
-
-        if (error) {
-          toast.error(error.message);
-          if (error.code === "email_not_confirmed") {
-            navigate(EmbeddedPaths.AuthEmailVerify);
-          }
-          return;
-        }
-
-        const interval = setInterval(() => {
-          if (authStatusRef.current === "unlocked") {
-            navigate(EmbeddedPaths.WalletHomeEmbeddedView);
```

Comment:
No redirect to home. The router takes care of this once `authStatus` is updated.